packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/__tests__/quanticFeedbackModal.test.js

```diff
@@ -0,0 +1,436 @@
+// @ts-ignore
+import {createElement} from 'lwc';
+import QuanticFeedbackModal from '../quanticFeedbackModal';
+
+const functionsMocks = {
+  submitFeedback: jest.fn(),
+};
+
+const selectors = {
+  errorMessage: '.error-message',
+  submitButton: '.feedback-modal-footer__submit',
+  successMesage: '.feedback-modal-body__success-message',
+  radioGroup: 'lightning-radio-group',
+  textarea: 'textarea',
+  detailsErrorMessage: '.feedback-modal-body__details-error-message',
+};
+
+const exampleValue = 'example value';
+const exampleLabel = 'example label';
+const exampleDetails = 'example details';
+/** @type {Array} */
+const exampleOptions = [{label: exampleLabel, value: exampleValue}];
+
+const errors = {
+  invalidArrayError: 'The options provided are not in a valid array.',
+  emptyArrayError: 'At least one option must be specified.',
+  missingLabelOrValueError:
+    'Each option requires a label and a value to be specified.',
+  missingSubmitFeedback: 'The submitFeedback property is not a valid function.',
+  invalidLabelTypeError: (label) =>
+    `The "${label}" label is not a valid string.`,
+  invalidValueTypeError: (value) =>
+    `The "${value}" value is not a valid string.`,
+};
+
+const defaultOptions = {
+  options: exampleOptions,
+  submitFeedback: functionsMocks.submitFeedback,
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-feedback-modal', {
+    is: QuanticFeedbackModal,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line @lwc/lwc/no-async-operation
```

Comment:
It would be nice to find the config line to put in the eslintrc to ignore that error in any `__tests__/*` file instead of having to have that line everywhere

Reply:
sure yeah 👌